diff -Naur qtwebengine-everywhere-src-5.11.1-original/src/core/core_module.pro qtwebengine-everywhere-src-5.11.1/src/core/core_module.pro
--- qtwebengine-everywhere-src-5.11.1-original/src/core/core_module.pro	2018-07-27 17:31:34.683537474 +0300
+++ qtwebengine-everywhere-src-5.11.1/src/core/core_module.pro	2018-07-27 17:33:36.792524395 +0300
@@ -54,7 +54,7 @@
 POST_TARGETDEPS += $$NINJA_TARGETDEPS
 
 
-LIBS_PRIVATE += -L$$api_library_path
+LIBS_PRIVATE += -L$$api_library_path -lexecinfo
 CONFIG *= no_smart_library_merge
 osx {
     LIBS_PRIVATE += -Wl,-force_load,$${api_library_path}$${QMAKE_DIR_SEP}lib$${api_library_name}.a
